### DISCUSSION TITLE
Remove/disable logging to file

### DISCUSSION THREAD

**Discussion by fullben** [JanWiemer/jacis#18] [Ideas]
The release version I tested (2.0.6) contains a logging configuration which causes the framework to write a log to file at runtime. Shouldn't it be the responsibility of the users of the framework to provide a logging config/decide on what they want to be logged?

**Comment by JanWiemer** [comment]
You are right! Logging Config should be up to the user of the library!
Added issue: https://github.com/JanWiemer/jacis/issues/24#issue-986713222
